magento/magento2
# feat: Add support for middle name in Cart addresses

Add support for middle name in Cart Address (billing, shipping)

### Description (*)
It seems the middle name is currently unsupported for the GraphQL implementation of the Cart addresses. The reason is unclear to me.

### Manual testing scenarios (*)
1. Create a Cart and populate the Billing and / or Shipping Address using a GraphQL mutation including a middle name
2. Check if the middle name is stored in the database
3. Check if the middle name is returned when getting the Cart through a GraphQL query.

### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#39088: feat: Add support for middle name in Cart addresses
